go/inst/instance_dao.go

```diff
@@ -804,6 +804,28 @@ func ReadTopologyInstanceBufferable(instanceKey *InstanceKey, bufferWrites bool,
 		}
 	}
 
+	// PseudoGTID allows for measurement of IO thread lag
+	if instance.UsingPseudoGTID {
+		waitGroup.Add(1)
+		go func() {
+			defer waitGroup.Done()
+			_, pseudoGtidText, err := FindLastPseudoGTIDEntry(instance, instance.RelaylogCoordinates, nil, false, &instance.Binlog_format)
```

Comment:
This is a relatively resource heavy operation. It's fine to run when you're in need to relocate a replica, but I wouldn't want to run this every 5 seconds as part of probing a server.